dask/dataframe/groupby.py

```diff
@@ -347,7 +347,7 @@ def wrapper(self, *args, **kwargs):
                 if not PANDAS_GT_150 and (
                     numeric_only is False
                     or numeric_only is not no_default
-                    and func.__name__ in ["corr", "cov", "cumprod", "cumsum"]
+                    and func.__name__ in ["cumprod", "cumsum"]
```

Comment:
`corr` and `cov` still don't support `numeric_only=False`, right? or do they now?

Reply:
No, this is about true only

Reply:
So then why are you removing `corr` and `cov` from this clause?

Reply:
False would error a few lines before this clause. This condition is only about true